sandpiper/processes/wps_resolve_rules.py

```diff
@@ -0,0 +1,134 @@
+from pywps import Process, LiteralInput, ComplexOutput, FORMATS
+from pywps.app.Common import Metadata
+import csv
+import json
+import logging
+import requests
+import os
+from decimal import Decimal
+from p2a_impacts.resolver import resolve_rules
+from p2a_impacts.utils import get_region, REGIONS
+
+
+LOGGER = logging.getLogger("PYWPS")
```

Comment:
Is `LOGGER` used anywhere in this program or are you waiting for `wps_tools` as well?

Reply:
I was waiting but I should probably throw some in just to have.